src/plugins/data-card-tool/data-card-tool.test.tsx

```diff
@@ -4,6 +4,7 @@ import { IToolApi } from "../../components/tools/tool-api";
 import { ToolTileModel } from "../../models/tools/tool-tile";
 import { defaultDataCardContent } from "./data-card-content";
 import { DataCardToolComponent } from "./data-card-tool";
+import { ModalProvider } from "@concord-consortium/react-modal-hook";
```

Comment:
By convention, `node_modules` imports are generally listed first, so move this import ⬆️.